clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -200,6 +200,105 @@ void CIRGenModule::emitGlobalFunctionDefinition(clang::GlobalDecl gd,
   }
 }
 
+mlir::Operation *CIRGenModule::getGlobalValue(StringRef name) {
+  mlir::Operation *global = mlir::SymbolTable::lookupSymbolIn(theModule, name);
+  if (!global)
```

Comment:
Er... what does this do?  This 'if' ends up being a do-nothing here, since it just returns 'global' anyway.